apps/server/test/lexical/server/code_intelligence/completion_test.exs

```diff
@@ -1,16 +1,16 @@
 defmodule Lexical.Server.CodeIntelligence.CompletionTest do
-  alias Lexical.Server.Project.Dispatch
-  alias Lexical.Server.CodeIntelligence.Completion
   alias Lexical.Project
   alias Lexical.Protocol.Types.Completion.Context, as: CompletionContext
   alias Lexical.Protocol.Types.Completion.Item, as: CompletionItem
   alias Lexical.RemoteControl
   alias Lexical.Server
+  alias Lexical.Server.CodeIntelligence.Completion
+  alias Lexical.Server.Project.Dispatch
   alias Lexical.SourceFile
 
   use ExUnit.Case
```

Comment:
Now we put the `use` before import?

Reply:
isn't that how it is?

Credo by default enforces use before _alias_, which sucks and is wrong, IMO. If I start doing strict module order, I'm going to definitely change the defaults.

Reply:
https://github.com/elixir-lsp/elixir-ls/pull/840

our rule is put the `use` at last. @scohen